_posts/2019-05-29-cockpit-195.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Cockpit 195
+author: lis
+date: 2019-05-29
+tags: cockpit linux
+slug: cockpit-195
+category: release
+summary: Firewall, storage and machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 195.
+
+### Simplified support for external TLS termination
+
+It's now easier to configure Cockpit's web server `cockpit-ws` to run
+behind a
+[TLS termination proxy](https://en.wikipedia.org/wiki/TLS_termination_proxy).
+If the proxy runs on the same machine, then `cockpit-ws` can be run with
+the new `--for-tls-proxy` option, which will adjust the allowed
+[Origins](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin)
+and
+[Content-Security-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+to http:// URLs. With this option, it's no longer necessary to
+explicitly configure
+[cockpit.conf](https://cockpit-project.org/guide/latest/cockpit.conf.5.html).
+
+### Firewall: More support for zones
+
+In addition to the support for adding services to specific zones in the
+last release, the firewall page now allows the adding and removing of
```

Comment:
"the" adding.. sounds a bit strange to me (but IANAENS)

Reply:
As a native speaker, I'd say "IANANES" :joy: